### DISCUSSION TITLE
NOT ABLE TO LOGIN TO MY SOCIALPHISH ACCOUNT IN KALI LINUX

### DISCUSSION THREAD

**Discussion by Ghost0101010121** [community/community#130529] [Other Feature Feedback, Questions, & Ideas]
I try and try over and over again and even made a token but nothing works plus on top of that keeps telling me authentication failed or......remote repository not found??? What does this all mean? I have manjaro and using a USB for Live System Kali Linux. Please help

**Comment by Rishi-Sudhakar** [comment]
Clear cache, setup all over again,
```
git credential-cache exit
```
(This will clear cache)
```
git config --global user.name "your_username"
git config --global user.email "your_email@example.com"
```
(Set your details correctly)

Setup authentication using SSH, then add repository url manually,

```
git remote set-url origin git@github.com:your_username/your_repository.git
```

**Reply by Ghost0101010121** [reply]
Okay thank you so much but where do I put these commands in? Konsole?? I use manjaro which is a pain and I don't know how to operate on this system, Its more complex for me as I like the linux version...thing is I just bought the Pinebook Pro 64. It uses manjaro ARM 64 and I know nothing about this lol. Please guide me step by step on how to also create the SSH and repository URL. Thank you
